trackintel/preprocessing/positionfixes.py

```diff
@@ -46,9 +48,11 @@ def generate_staypoints(positionfixes,
     
     Returns
     -------
-    (GeoDataFrame, GeoDataFrame)
-        Tuple of (positionfixes, staypoints). Positionfixes is the original GeoDataFrame with 
-        a new column 'staypoint_id'. 
+    ret_pfs: GeoDataFrame (as trackintel positionfixes)
```

Comment:
Good fix! I wasn't aware that you have to list all the different return values separately.